tests/integration/graviton_test.py

```diff
@@ -513,6 +513,7 @@ def blackbox_test_runner(
     inputs, predicates = Invariant.inputs_and_invariants(
         scenario, exec_mode, raw_predicates=True
     )
+    assert predicates, "Must configure >= 1 predicate"
```

Comment:
This proves a graviton bug. It should assert (by default) that predicates are actually provided.

Reply:
I've incorporated this change in #605. Here's the commit that prove it: https://github.com/algorand/pyteal/pull/605/commits/2dc41daf7875c90a536c0bde577b9167b4d4f2fa

Reply:
@tzaffi Thanks for bringing in the defensive check.  I'll close this PR.